pkg/notification/storage/v2/sql/subscription/select_subscription_v2_any.sql

```diff
@@ -1,11 +1,13 @@
 SELECT
-    id,
-    created_at,
-    updated_at,
-    disabled,
-    source_types,
-    recipient,
-    name
+    sub.id,
+    sub.created_at,
+    sub.updated_at,
+    sub.disabled,
+    sub.source_types,
+    sub.recipient,
+    sub.name
 FROM
-    subscription
+    subscription sub
+JOIN environment_v2 env
+     ON sub.environment_id = env.id
```

Comment:
You are joining the environment table, but you are not using any field from that table.
We joined the query to get the environment name. 

https://github.com/bucketeer-io/bucketeer/blob/c3140e135139371a614fa8e46bd0c5f0bf7a3484/pkg/account/storage/v2/sql/api_key_v2/select_api_key_v2.sql#L9